app/helpers/application_helper.rb

```diff
@@ -6,4 +6,14 @@ def identifier_types
   def percentage(ratio)
     "#{(ratio * 100).round}%"
   end
+
+  def github_search_enabled?
+    ENV['GITHUB_ORGANIZATION'].present?
+  end
+
+  def github_search_url(split)
+    return nil unless github_search_enabled?
```

Comment:
One might argue that this could `raise`, since the existence of `github_search_enabled?` implies that you shouldn't call this method unless that returns true.

Reply:
Alternatively, this could be a nullable facet of the split model:

`@split.github_search_url`

And the null/presence could be checked via a predicate method -- `@split.github_search_url?`